aircraftlib/aircraft.capnp.go

```diff
@@ -27,11 +26,18 @@ func (s Zdate) MarshalJSON() (bs []byte, err error) { return }
 
 type Zdate_List C.PointerList
 
-func NewZdateList(s *C.Segment, sz int) Zdate_List { return Zdate_List(s.NewUInt32List(sz)) }
+func NewZdateList(s *C.Segment, sz int) Zdate_List { return Zdate_List(s.NewCompositeList(8, 0, sz)) }
```

Comment:
@zombiezen Hi Ross, Thanks for this pull request.  I don't see included the change to capnpc-go/capnpc-go.go that makes write NewCompositeList() instead of NewUInt32List(). Is there an additional commit that does this?